search-website-functions-v4/api-v4/local.settings.json.rename

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this file supposed to be in the changes?

Reply:
So -- it's in the original, I think, because local.settings.json is explicitly excluded in the .gitignore file in the folder. The doc topics that this sample is used in don't have you building locally, so it doesn't really get used. On the other hand, I found it nice to have when I did build and run locally -- all I had to do was rename the file.

Reply:
I would like to keep it - unless a customer creates the function app, they don't know how to add environment variables.